Handle issue when ip is a bogon (#8)
ipinfo/csharp: README.md

    .Build();
```

### Bogon filtering

> It is suggested to check whether the ip is a bogon or not. `Bogon` property of `IPResponse` object can be used to check if ip address is a bogon.

Reviewer: ```suggestion
The `Bogon` property of the `IPResponse` object can be used to check if an IP address is a bogon.
```
